packages/govuk-frontend/src/govuk/components/error-summary/_index.scss

```diff
@@ -14,15 +15,14 @@
   }
 
   .govuk-error-summary__title {
-    @include govuk-font($size: 24, $weight: bold);
+    @include govuk-typography-responsive($size: 24);
+    @include govuk-typography-weight-bold;
```

Comment:
Can we drop all these `font-weight: 700` lines for every heading?

Unless the browser default `font-weight: bold` (same as 700) has issues?

```suggestion
```

Reply:
The weight mixins are tied to the variables `$govuk-font-weight-bold` and `$govuk-font-weight-regular`. I'd rather maintain that relationship.

There's a question this raises for me on the value of those mixins but that's a not today problem.

Reply:
IIRC, there are a few things we've explored in the past that, if we ever did them, would benefit from using the mixins:

- only enabling font smoothing for bold text
- using `font-size-adjust`, which has [different values for bold vs normal text](https://github.com/alphagov/govuk_frontend_toolkit/pull/271)

It'd also allow us to e.g. introduce a mid weight if we wanted to.

Arguably all of these are premature optimisations but I don't think there's enough justification to remove them now.